include/n2n_define.h

```diff
@@ -63,6 +63,10 @@
 #define N2N_COMPRESSION_ID_ZSTD		3	/* set if '-z2' cli option is present, available only if compiled with zstd lib */
 #define ZSTD_COMPRESSION_LEVEL		7	/* 1 (faster) ... 22 (more compression) */
 
+/* Federation indicators */
+#define IS_FEDERATION        1
```

Comment:
Can you please define an enum datatype with all the possible states instead of using #defines ?

Reply:
I've added an enum datatype with the two values